content/chat/moderation/index.textile

```diff
@@ -37,12 +37,18 @@ There are a plethora of moderation options available on the market, from simple
 
 Ably provides direct integrations between your chat room and moderation providers, to give you access to powerful moderation platforms with minimal code and setup. If a provider you are looking for is not listed, please get in touch!
 
-Alternatively, you might have a custom solution you wish to integrate with, or a provider that Ably doesn't yet directly support. If this is the case, Ably offers a custom option, where you can utilize serverless functions such as AWS Lambda to reach out to your own infrastructure to moderate chat messages.
+Alternatively, you might have a custom solution you wish to integrate with, or a provider that Ably doesn't yet directly support. If this is the case, "Ably offers a custom option":/docs/chat/moderation/custom, where you can utilize serverless functions such as "AWS Lambda":/docs/chat/moderation/custom/lambda to reach out to your own infrastructure to moderate chat messages.
 
 h3(#hive). Hive
 
-Hive provide automated content moderation solutions. The first of these is the "model only":https://hivemoderation.com/text-moderation solution, which provides access to a powerful ML model that takes content and categorises it against various criteria, for example, violence or hate speech. For each classification, it also provides an indication of the severity of the infraction. Using this information, you can determine what level of classification is appropriate for your chat room and filter / reject content accordingly.
+Hive provide automated content moderation solutions. The first of these is the "model only":https://hivemoderation.com/text-moderation solution, which provides access to a powerful ML model that takes content and categorises it against various criteria, for example, violence or hate speech. For each classification, it also provides an indication of the severity of the infraction. Using this information, you can determine what level of classification is appropriate for your chat room and filter / reject content accordingly. "Read more about how to add model moderation":/docs/chat/moderation/direct/hive-model-only.
```

Comment:
It feels weird to have a sentence on the end that is a hyperlink (it doesn't quite read right) - perhaps something more like:

`Click "here":/docs/chat/moderation/direct/hive-model-only to read more about how to add model-only moderation with Hive`

Reply:
I'm avoiding "click here" type of links for accessibility but maybe something like `"Read more about how to [add model moderation]"`

Or the link as a paragraph?

"Click here" looks good though.

Reply:
I agree.

I've tried the bullet point link on next paragraph and it looks weird.

What if I remove those new links and change the existing ones that point to hive to point to our docs? The other pages link to the relevant hive docs anyway.

Reply:
Pushed this to share what it looks like